ui/pages/settings/experimental-tab/experimental-tab.component.js

```diff
@@ -261,7 +261,17 @@ export default class ExperimentalTab extends PureComponent {
               id="select-theme"
               options={themesOptions}
               selectedOption={theme}
-              onChange={async (newTheme) => setTheme(newTheme)}
+              onChange={(newTheme) => {
+                this.context.trackEvent({
+                  category: 'Settings',
+                  event: 'Theme Change',
+                  properties: {
+                    theme_selected: newTheme,
+                    legacy_event: true,
```

Comment:
[REQ] remove legacy_event, this is reserved for events that were a part of our old schema.